packages/vscode-extension/package.nls.json

```diff
@@ -395,6 +395,8 @@
     "teamstoolkit.tutorials.commandAndResponse.label": "Respond to chat commands in Teams",
     "teamstoolkit.tutorials.connectApi.detail": "Connect to an API with authentication support using TeamsFx SDK.",
     "teamstoolkit.tutorials.connectApi.label": "Connect to an API",
+    "teamstoolkit.tutorials.addAzureSql.label": "Add Azure SQL Database",
+    "teamstoolkit.tutorials.addAzureSql.detail": "Add an Azure SQL Database to your project",
```

Comment:
<img width="777" alt="image" src="https://user-images.githubusercontent.com/11220663/205268933-e696987b-54e9-446d-9d3d-280882a6d499.png">

Can we use strings from this [design file](https://www.figma.com/file/XmE23PoGtrVcapAGABFiVp/%E3%80%90VSC%E3%80%91Teams-Toolkits-optimize?node-id=522%3A102466&t=nKaNg97kMezFALR6-1)?

Reply:
thanks, updated